libfwupd/fwupd-bios-attr.c

```diff
@@ -477,6 +518,12 @@ fwupd_bios_attr_to_variant(FwupdBiosAttr *self)
 			      "{sv}",
 			      FWUPD_RESULT_KEY_BIOS_ATTR_TYPE,
 			      g_variant_new_uint64(priv->kind));
+	if (priv->attr_id != NULL) {
+		g_variant_builder_add(&builder,
+				      "{sv}",
+				      FWUPD_RESULT_KEY_BIOS_ATTR_ID,
```

Comment:
I think FWUPD_RESULT_KEY_APPSTREAM_ID works here, no?

Reply:
That was my original thought. But if you want to share with security attr you need to have two different key names.

Reply:
Bleh, indeed. Sorry.